### PR TITLE
Animations and styling

### PR DESCRIPTION
**Pull request by Lnola**
This PR focuses on adding animations and fixing styling issues.

Issues fixed:
- scaling of register form on smaller screens so padding stays about the same
- changed the way profile sidebar is toggled on and off
- fixed scrollbars showing up where they shouldn't and added custom ones

Animations are simple, done to liven up the app without distracting from the content. They are mostly transitions done with the Vue `<Transition>` tag

Closes: #80

### CODE REVIEW

File: frontend/src/components/profile/ProfileSidebar.vue [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-        <tag-list @select-tag="selectTag" :tags="tags" class="user-profile-tags" centered small />
-        <chi-button @click="authStore.signOut(router)" class="user-profile-button" inline>Sign out</chi-button>
+  <aside>
+    <transition name="slide">
+      <div v-if="active" class="profile-container">
+        <div class="profile-wrapper">
+          <img src="@/assets/img/profile.svg" alt="profile" class="profile-avatar" />
+          <user-form :inputs="formInputs" class="profile-form" disabled />
+          <tag-list @select-tag="selectTag" :tags="tags" class="profile-tags" centered small />
+          <chi-button @click="authStore.signOut(router)" class="profile-button" inline>Sign out</chi-button>
```
Comment (sandje00) [inline frontend/src/components/profile/ProfileSidebar.vue:9]: On smaller screens Sign out button is partially cut out, so can you, please, resolve it?

Comment (Lnola) [reply frontend/src/components/profile/ProfileSidebar.vue:9]: Them pesky giant taskbars/docks

File: frontend/src/components/tags/TagSelect.vue [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+        <h1 class="select-title">Select your favorite music genres</h1>
+        <h3 class="select-subtitle">
+          Tell us your favorite music genres so you see the rooms you want to visit first!
+        </h3>
+      </div>
+
+      <tag-list @select-tag="selectTag" class="select-tags" :tags="tags"></tag-list>
+      <chi-button @click="handleSaveSelected" class="select-skip" :disabled="!areAnySelected" inline
+        >Continue</chi-button
+      >
```
Comment (abasic) [inline frontend/src/components/tags/TagSelect.vue:14]: Why is this in a new line?

Comment (Lnola) [reply frontend/src/components/tags/TagSelect.vue:14]: Because the formatter is so very smart 😄

**Review by abasic — APPROVED** [review]

**Pull request merged**